kubeflow/common/prototypes/centraldashboard.jsonnet

```diff
@@ -3,7 +3,7 @@
 // @description centraldashboard Component
 // @shortDescription centraldashboard
 // @param name string Name
-// @optionalParam image string gcr.io/kubeflow-images-public/centraldashboard:v20190223-v0.4.0-rc.1-152-g1de09b7a Image for the central dashboard
+// @optionalParam image string gcr.io/kubeflow-images-public/centraldashboard:v20190301-v0.4.0-rc.1-183-g07eecd2f Image for the central dashboard
```

Comment:
Just a question, does this comment need to be updated with every subsequent release? Wouldn't a user typically just want to deploy the **gcr.io/kubeflow-images-public/centraldashboard:latest** or search for their specific tag during development?

Reply:
So we definitely don't like pushing to `:latest`, though there is a make recipe for it (@jlewi can possibly elaborate on why that is). But no we don't generally make an update everytime. But because the last 2 images were broken (bower deps were missing, and wrong port was being used). This release fixes both those issues.